components/inputs/test/input-text.visual-diff.js

```diff
@@ -0,0 +1,53 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-input-text', function() {
+
+	const visualDiff = new VisualDiff('input-text', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/components/inputs/test/input-text.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	after(() => browser.close());
+
+	[
+		'basic',
+		'email',
+		'number',
+		'password',
+		'search',
+		'tel',
+		'url',
+		'disabled',
+		'placeholder',
+		'placeholder-disabled',
+		'invalid',
+		'invalid-disabled',
+		'aria-invalid',
+		'aria-invalid-disabled',
+		'override-height',
+		'override-padding',
+		'override-placeholder'
+	].forEach((name) => {
+		it(name, async function() {
```

Comment:
Not an issue, but just something to watch out for... if these two tests were executed in the opposite order, I think the non-focused screenshot would have focus style since focus is not reset between tests.